README.rst

```diff
@@ -63,6 +63,10 @@ for the parameters passed in at execution time. It acheive this by inserting a
 cell after the tagged cell. If no cell is tagged with ``parameters`` a cell will
 be inserted to the front of the notebook.
 
+Additionally, you can rerun notebooks through papermill and it will reuse the
+parameter cell it injected in the prior run. When a papermill generated notebook
+executes it will instead replace the ``parameters`` tagged cell.
```

Comment:
I had to re-parse this mentally multiple times to understand.

What I think might have been confusing for me is that the original model for papermill was to replace the parameters cell to begin with. Then if I recall correctly we started inserting another cell so that we wouldn't have to parse the cell to do a perfect replacement (is that right)?

How does this setup ensure consistency?

Reply:
Correct. The difference now is that we know the parameter cell is a generated element and perfect replacement is not necessary. Before we had defaults in the `parameters` cell generated by users which the replacement call would erase if it didn't have a new value for that name.

Having thought about the changes and the complexity for a couple days. I think maybe the PR should change to be simpler. Maybe this could be the new rule: If a `default parameters` cell exists, replace the `parameters` cell. If it doesn't exist, change `parameters` to `default parameters` and insert a new `parameters` cell. @rgbkrk What do you think about that model? I think it's easy to describe and still makes rerunning notebooks have a nicer output format.